prime-router/src/test/kotlin/validation/MarsOtcElrOnboardingValidatorTests.kt

```diff
@@ -0,0 +1,23 @@
+package gov.cdc.prime.router.validation
+
+import assertk.assertThat
+import assertk.assertions.isFalse
+import gov.cdc.prime.router.ActionLogger
+import gov.cdc.prime.router.fhirengine.utils.HL7Reader
+import org.junit.jupiter.api.Test
+
+class MarsOtcElrOnboardingValidatorTests {
+
+    private val validator = MarsOtcElrOnboardingValidator()
+
+    @Test
+    fun `test can detect an invalid RADxMARS message`() {
+        val sampleMessageInputStream =
+            this.javaClass.classLoader.getResourceAsStream("validation/marsotcelr/sample_2.hl7")
```

Comment:
Might be helpful to rename the file something more descriptive.

Reply:
Unfortunately, I don't know the details of the file, beyond the fact that it was used to fail the other test, so I don't know what to name it to differentiate it from sample_1.hl7 since that one also seems to fail validation. Decided to use my own files.